web-components/src/components/not-found/index.tsx

```diff
@@ -50,78 +46,47 @@ const useStyles = makeStyles((theme: Theme) => ({
       left: `-${theme.spacing(1.5)}`,
     },
   },
-  fourOFour: {
-    fontFamily: theme.typography.h1.fontFamily,
-    fontStyle: 'normal',
-    fontWeight: 900,
-    [theme.breakpoints.up('sm')]: {
-      fontSize: theme.spacing(48),
-      lineHeight: theme.spacing(42.75),
-      marginBottom: theme.spacing(8),
-    },
-    [theme.breakpoints.down('sm')]: {
-      fontSize: theme.spacing(28),
-      lineHeight: theme.spacing(30.75),
-      marginBottom: theme.spacing(2),
-    },
-
-    textAlign: 'center',
-    letterSpacing: theme.spacing(1),
-    textTransform: 'uppercase',
-    color: theme.palette.secondary.dark,
-  },
-  h1: {
-    fontWeight: 900,
-    lineHeight: '130%',
-    color: theme.palette.primary.contrastText,
-    marginBottom: theme.spacing(4.25),
-  },
-  text: {
-    lineHeight: '150%',
-    fontSize: theme.spacing(4.5),
-    marginBottom: theme.spacing(6.5),
-  },
-  button: {
-    textTransform: 'uppercase',
-    height: theme.spacing(15),
-    lineHeight: theme.spacing(5.75),
-    [theme.breakpoints.down('sm')]: {
-      padding: `${theme.spacing(3)} ${theme.spacing(7.5)}`,
-      fontSize: theme.spacing(4.5),
-      height: 'unset',
-    },
-    [theme.breakpoints.up('sm')]: {
-      padding: `${theme.spacing(4.5)} ${theme.spacing(12.5)}`,
-      fontSize: theme.spacing(4.5),
-    },
-  },
 }));
 
 const NotFound = ({ img, home = '/' }: NotFoundProps): JSX.Element => {
   const classes = useStyles();
 
   return (
-    <div className={classes.root}>
+    <Box sx={{ backgroundColor: 'primary.main' }}>
       <div className={classes.container}>
         <div className={classes.image}>{img}</div>
-        <div className={classes.fourOFour}>404</div>
```

Comment:
much of the work in this PR was done through searching for elements I knew needed to be replaced, and I tried to find as many places like this as I could where we're using `makeStyles` applied to normal JSX elements, but I probably missed several. If you run into these in the future, they should all (with a few small exceptions like text formatted as code) be replaceable by the new elements.